### PR TITLE
Make minor fixes to Celery config

### PR DESCRIPTION
**Pull request by ddohler**
This fixes a few minor issues I ran into with Celery tasks. I'm making this as a separate PR since it sounds like some other folks may be working on Celery tasks soon too.

### CODE REVIEW

**Comment by kshepard** [comment]
+1, looks good to me. @sharph, want to take a quick look?

File: deployment/ansible/roles/driver.celery/templates/upstart-celery.conf.j2 [hunk 1]
```diff
@@ -21,7 +21,7 @@ pre-start script
 end script
 
 exec /usr/bin/docker run \
-  --name driver-app \
+  --name driver-celery \
```
Comment (sharph) [inline deployment/ansible/roles/driver.celery/templates/upstart-celery.conf.j2]: The cronjob may need to be altered to match this.

**Comment by sharph** [comment]
:+1:

File: deployment/ansible/roles/driver.celery/templates/upstart-celery.conf.j2 (hunk 1 above)
Comment (ddohler) [inline deployment/ansible/roles/driver.celery/templates/upstart-celery.conf.j2]: Good call, fixed.

**Pull request merged**